packages/xo-web/src/xo-app/vm-import/esxi/esxi-import.js

```diff
@@ -87,6 +89,7 @@ class EsxiImport extends Component {
       sslVerify: !skipSslVerify,
       stopOnError,
       stopSource,
+      template: template?.id,
```

Comment:
```suggestion
      template: template.id,
```


Should not be `undefined` if `template` is required